firmware/t3b1/releases.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Is this safe to add empty? Its just for consistency.

Reply:
Should be safe.